app/assets/javascripts/vocab_bar.js

```diff
@@ -1,19 +1,29 @@
 $(document).ready(function() {
-  function hideVocabSheetOnMobile() {
-    var bar = $('.vocab_sheet_bar');
-    var windowWidth = $(window).width();
-    if (windowWidth < 640 || (bar.length && bar.find('.vocab_sheet_bar_item').length === 0)) {
-      bar.hide();
-      $('.not_sticky_footer').removeClass('vocab_sheet_background');
-    } else if (windowWidth > 639 &&  bar.length) {
-      bar.show();
-      $('.not_sticky_footer').addClass('vocab_sheet_background');
+  if ($('.vocab-sheet__bar').length > 0) {
+    var bar = $('.vocab-sheet__bar');
+    var searchResultBanner = $('.search-result-banner');
+    var offset = searchResultBanner.offset().top;
+    var bottom = offset + searchResultBanner.outerHeight(true);
+    var baseMargin = 24; // 1.5rem
+    var initialOffset = bottom + baseMargin;
+    var footerTop = $('.sticky_footer').offset().top;
+
+    function positionVocabBar() {
+      var windowScroll = $(window).scrollTop();
+      var barBottom = bar.offset().top + bar.outerHeight(true) + baseMargin;
+
+      if (windowScroll < bottom) {
+        bar.css({top: initialOffset - offset});
```

Comment:
Can this same thing be achieved with sticky css positioning? Or am I mis reading what it does?

Reply:
The short answer is yes, this is essentially sticky positioning, however we unfortunately can't use sticky because of the IE support being nil: https://caniuse.com/#feat=css-sticky

I'll look into your suggestion for using `fixed` though.

Reply:
An additional follow up to this. We tried implementing the foundation sticky plugin but theres something in the existing styling thats preventing it from working. We've opted to backlog that particular implementation for now and get this over the line.